desktop/ponyprog.desktop

```diff
@@ -1,12 +1,12 @@
 [Desktop Entry]
-Version=3.0.0
+Version=3.0
```

Comment:
This should be `1.1` as its the version of desktop spec. See: https://standards.freedesktop.org/desktop-entry-spec/latest/ar01s06.html